bin/run

```diff
@@ -29,13 +30,16 @@ class Exercise
   private
 
   def format_output(positions)
-    positions.join(' ') << "\n"
+    positions.sort.join(' ') << "\n"
   end
 
   def calculate_possible_positions(line)
     ChessPieceMove.new(line, figure_move_class).possible_positions
   end
-
 end
 
-Exercise.new(Figure::Queen).call
\ No newline at end of file
+if Support::FetchAClass.new(ARGV[1]).class_valid?
```

Comment:
is it possible to rewrite that code, so from the point of view for Exercise class the only public method on `Support::FetchAClass` is `figure_move_class` ?